src/Sylius/Bundle/CoreBundle/Resources/views/Email/layout.html.twig

```diff
@@ -0,0 +1,28 @@
+{% block body %}
+    {% autoescape false %}
+        {% set logo = channel.hostname is not null ? 'http://' ~ channel.hostname ~ asset('assets/shop/img/logo.png') : absolute_url(asset('assets/shop/img/logo.png')) %}
```

Comment:
Logo is taken from shop namespace, I'm not sure if it is coupling CoreBundle with ShopBundle

Reply:
It is just an asset. Perhaps we should move some image to CoreBundle as well, but for it should will allow the application to be run without Shop bundle.